chapters/ch26.md

```diff
@@ -1,26 +1,34 @@
 | Audio clip  | Loud background noise | User spoke quickly | Far from microphone |   Comments |
 | ----------: | :-------------------: | :----------------: | :-----------------: |  :-------: |
 | 1           |         x             |                    |                     | Car noise |
 | 2           |         x             |                    |          x          | Restaurant noise |
 | 3           |                       |       x            |          x          | User shouting across living room? |
 | 4           |         x             |                    |                     | Coffeeshop |
 | % of total  |        75%            |      25%           |         50%         |
+->
+| Clip âm thanh  | Nhiễu nền lớn | Nguời dùng nói nhanh | Xa micro |   Bình luận |
+| ----------: | :-------------------: | :----------------: | :-----------------: |  :-------: |
+| 1           |         x             |                    |                     | Tiếng ồn xe hơi |
+| 2           |         x             |                    |          x          | Tiếng ồn nhà hàng |
+| 3           |                       |       x            |          x          | Nguời dùng la hét khắp phòng khách? |
+| 4           |         x             |                    |                     | Quán cà phê |
+| % tổng thể  |        75%            |      25%           |         50%         |
 
 In this example, you might realize that your algorithm is having a particularly hard time with training examples that have a lot of background noise. Thus, you might focus on techniques that allow it to better fit training examples with background noise.
 ->
-
+Trong ví dụ này, bạn có thể nhận ra rằng thuật toán của bạn đang có một thời gian đặc biệt khó khăn với các mẫu huấn luyện có nhiều nhiễu nền. Do đó, bạn có thể tập trung vào các kỹ thuật cho phép nó khớp hơn với các ví dụ đào tạo với nhiễu nền.
 You might also double-check whether it is possible for a person to transcribe these audio clips, given the same input audio as your learning algorithm. If there is so much background noise that it is simply impossible for anyone to make out what was said, then it might be unreasonable to expect any algorithm to correctly recognize such utterances. We will discuss the benefits of comparing your algorithm to human-level performance in a later section.
 ->
-
+Bạn cũng có thể kiểm tra kỹ xem một người có thể sao chép các clip âm thanh này hay không, đưa ra âm thanh đầu vào giống như thuật toán learning của bạn. Nếu có quá nhiều nhiễu nền mà đơn giản là không ai có thể phát hiện ra những gì đã nói, thì có thể sẽ không hợp lý khi mong đợi bất kỳ thuật toán nào nhận ra chính xác những phát ngôn đó. Chúng ta sẽ thảo luận về lợi ích của việc so sánh thuật toán của bạn với hiệu suất ở cấp độ con người trong phần sau.
```

Comment:
"thuật toán learning" -> "thuật toán học"

"human-level performance" ở chương sau có chỗ được dịch là "hiệu quả cấp con người".

Từ "performance" mình cũng đã dịch là "hiệu quả" từ đầu rồi.